javascript-view-swapping/main.js

```diff
@@ -0,0 +1,28 @@
+const $tabContainer = document.querySelector('.tab-container');
+const $tab = document.querySelectorAll('.tab');
+const $view = document.querySelectorAll('.view');
+
+$tabContainer.addEventListener('click', function (e) {
+  if (e.target.matches('.tab')) {
+    for (let i = 0; i < $tab.length; i++) {
+      if (e.target === $tab[i]) {
+        e.target.className = 'tab active';
+      }
+      if ($tab[i] !== e.target) {
+        $tab[i].className = 'tab';
+      }
+    }
+    const $data = e.target.dataset.view;
+    console.log($data);
```

Comment:
Delete this console log! It's only for testing / debugging purposes. Unless we specifically show a console log in the example output